src/controllers/wallet/tx-proposal.controller.js

```diff
@@ -117,9 +119,9 @@ async function signAndPush(req, res) {
   }
 
   const { txHex } = req.body;
-  const signatures = req.body.signatures || [];
+  const signatures = req.body.signatures.slice(0, req.wallet.multisig.minSignatures) || [];
```

Comment:
The MultiSig implementation on the fullnode only checks the minimum signatures and rejects the transaction since the extra signatures remain on stack when validating.
This slice is meant to cut any signatures the user sends above the minimum so this error does not happen.